src/js/components/DataTable/stories/ColumnSize.js

```diff
@@ -0,0 +1,196 @@
+  { property: 'paid', header: 'Paid', size: 'xsmall', align: 'end' },
+];
+
+const columnsRelativeSize = [
+  { property: 'name', header: 'Name', size: '1/2' },
+  { property: 'location', header: 'Location', size: '1/4' },
+  { property: 'date', header: 'Date', size: '1/4' },
+];
+
+const columnsAbsoluteSize = [
+  { property: 'name', header: 'Name', size: '600px' },
+  { property: 'location', header: 'Location', size: '200px' },
+  { property: 'date', header: 'Date', size: '200px', align: 'end' },
+  { property: 'percent', header: 'Percent', size: '100px', align: 'end' },
+  { property: 'paid', header: 'Paid', size: '100px', align: 'end' },
+];
+
+const columnsDefault = [
+  { property: 'name', header: 'Name' },
+  { property: 'location', header: 'Location' },
+  { property: 'date', header: 'Date', align: 'end' },
+  { property: 'percent', header: 'Percent', align: 'end' },
+  { property: 'paid', header: 'Paid', align: 'end' },
+];
+
+const Example = () => (
+  <Grommet theme={grommet}>
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3"> Default DataTable</Heading>
+      <DataTable
+        columns={columnsDefault}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3">Theme Column Sizes</Heading>
+      <DataTable
+        columns={columnsThemeSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3">Absolute Column Sizes</Heading>
+      <DataTable
+        columns={columnsAbsoluteSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+
+    <Box fill="horizontal" pad="medium">
+      <Heading level="3">Relative Column Sizes</Heading>
+      <DataTable
+        columns={columnsRelativeSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        border={{
+          color: 'border',
+          side: 'vertical',
+          size: '1px',
+        }}
+      />
+    </Box>
+  </Grommet>
+);
+
+const ExampleResizable = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="medium">
+      <Heading level="3">Large table with resizeable</Heading>
+      <DataTable
+        columns={columnsThemeSize}
+        data={DATA}
+        step={10}
+        primaryKey={false}
+        resizeable
+      />
+    </Box>
+
+    <Box align="center" pad="medium">
+      <Heading level="3">Small table with resizeable</Heading>
+      <DataTable
+        columns={columnsReize}
+        data={DATA}
+        step={10}
```

Comment:
I don't think we are using `step` in those examples, let's remove it.

Reply:
Done!